tests/queries/0_stateless/00498_bitwise_aggregate_functions.reference

```diff
@@ -2,3 +2,11 @@
 1	[1,5,9,13,17]	29	1	17
 2	[2,6,10,14,18]	30	2	18
 3	[3,7,11,15,19]	31	3	19
+0       [0,-4,-8,-12,-16]       -4      0       0
```

Comment:
Please regenerate these expected results, CI job "Fast Test" currently reports an error. You can test locally with `PATH=$PATH:build/programs/ tests/./clickhouse-test 00498_bitwise_`.

Reply:
(Tried to fix them by myself but I don't have permissions to push into your fork of ClickHouse.)

Reply:
Thanks! Done.